rpmio/rpmpgp.h

```diff
@@ -266,6 +266,8 @@ typedef enum pgpHashAlgo_e {
     PGPHASHALGO_SHA384		=  9,	/*!< SHA384 */
     PGPHASHALGO_SHA512		= 10,	/*!< SHA512 */
     PGPHASHALGO_SHA224		= 11,	/*!< SHA224 */
+    PGPHASHALGO_GOST12_256	= 100,	/*!< GOST R 34.11-2012 256 */
+    PGPHASHALGO_GOST12_512	= 101,	/*!< GOST R 34.11-2012 512 */
```

Comment:
Is there a reason we jump so many values?

Reply:
It should match the hash algorithm id used in pgp.

12 is SHA3-256, 14 is SHA3-512. 100-110 is reserved for Private/Experimental algorithms.

See https://tools.ietf.org/html/draft-ietf-openpgp-rfc4880bis-05

Reply:
(I'd actually prefer to use official values. Is the GOST project in contact with the pgp folks?)

Reply:
@mlschroe What is "the GOST project"? I am not aware of any attempts to include GOST to OpenPGP RFC, but maybe someone tried, I do not know. We in ROSA with @StZhukov first wanted to implement signing RPM payload with GOST as it is signed with gpg, but then we found out that OpenPGP RFC does not support GOST, so implementing it does not make much sense. I would not expect GOST algos to appear in the RFC in the nearest future. Because of all this I would suggest to use values >110 here, which are not specified in the RFC and so will never become used by other algorithms.

Reply:
Dunno, I thought something like http://gostcrypt.github.io/